history/characters/34000_murloc.txt

```diff
@@ -17559,19 +17559,6 @@ murloc143={
 	621.6.17={ birth=yes trait=creature_murloc }
 	700.3.18={ death=yes }
 }
-
-cookie={
-	name=Cookie
-	culture=murloc religion=water_deities
-	trait=education_martial_3 trait=brave trait=patient trait=greedy 
-	583.4.10={ birth=yes trait=creature_murloc }
-	598.3.17={
-		employer=5811 #Edwin VanCleef
-		religion=masonry
-	}
-	655.9.22={ death=yes }
-}
-
```

Comment:
Why did you remove Cookie?

Reply:
1.  Tarian becomes Martial 4 in a later bookmark when he becomes a Paladin. Feel free to give it to him sooner, I felt his backstory meant he gained martial abilities later in life after becoming a Paladin. But idc.
2. Durnack is listed twice, the more recent bookmark he is 18, not 27. I'm assuming the patreon updated what he wanted for his character in the more recent entry. Also he is supposed to have Herculean in the most recent update and no longer has Genius.
3. I don't know why he adopts Al'akiric, but I gave him Illidari at 603.1.1
4. I made a mistake and forgot to include "lifestyle_" in front of herbalist.

Reply:
We should probably make a rule about making lore characters patreon characters.
Imagine someone asking to add Uther as a patreon.
I know their significance differs but the point still stands.

Reply:
I added the `lifestyle_` in front of `herbalist` for cookie. But we have to decide what to do with this character, maybe ask the patreon and let him for now ?

Reply:
> 1. Tarian becomes Martial 4 in a later bookmark when he becomes a Paladin. Feel free to give it to him sooner, I felt his backstory meant he gained martial abilities later in life after becoming a Paladin. But idc.

that's fine, but iirc you don't get any events to get an education if you don't already have one after becoming an adult. nbd as he's a courtier, but might be better to start him off at Martial 1 or 2 if backstory progression is your intent.

> 2. Durnack is listed twice, the more recent patreon update he is 18, not 27. I'm assuming the patreon updated what he wanted for his character in the more recent entry. Also he is supposed to have Herculean in the most recent update and no longer has Genius.

seems he has the strong trait instead of Herculean, checked all start dates.

> 3. I don't know why he adopts Al'akiric, but I gave him Illidari at 603.1.1

Yeah real bizzare. Paused game on start date, still Al'Akiric for me in 605 and 603

> 4. I made a mistake and forgot to include "lifestyle_" in front of herbalist.

looks good